src/index.ts

```diff
@@ -55,8 +55,8 @@ const waitForPipeline = async (pipelineName: string, pipelineExecutionId: string
         core.error(`Pipeline '${pipelineName}' stopped.`);
         return false;
       case PipelineExecutionStatus.Superseded:
-        core.error(`Pipeline '${pipelineName}' was superseded.`);
-        return false;
+        core.warning(`Pipeline '${pipelineName}' was superseded. Skipping rest of the execution.`);
+        return true;
```

Comment:
@snowiow this might still show Green in the GHA itself after throwing a warning. Not sure, how can we show skip sign there 🤔

Reply:
I think thats not possible because the action already ran and the skip sign is only shown if the action did not run at all